API/models/schemas/application_schema.json

```diff
@@ -0,0 +1,133 @@
+          {
+            "type": "object",
+            "properties": {
+              "namespace_kube": {
+                "type": "string"
+              },
+              "category": {
+                "type": "string"
+              },
+              "type": {
+                "type": "string"
+              },
+              "name": {
+                "type": "string"
+              },
+              "children": {
+                "type": "array",
+                "items": [
+                  {
+                    "type": "object",
+                    "properties": {
+                      "category": {
+                        "type": "string"
+                      },
+                      "type": {
+                        "type": "string"
+                      },
+                      "name": {
+                        "type": "string"
+                      },
+                      "children": {
+                        "type": "array",
+                        "items": [
+                          {
+                            "type": "object",
+                            "properties": {
+                              "pvc": {
+                                "type": "string"
+                              },
+                              "children": {
+                                "type": "array",
+                                "items": [
+                                  {
+                                    "type": "object",
+                                    "properties": {
+                                      "pv-type": {
+                                        "type": "string"
+                                      },
+                                      "pv-path": {
+                                        "type": "string"
+                                      },
+                                      "pv-ip": {
+                                        "type": "string"
+                                      }
+                                    },
+                                    "required": [
+                                      "pv-type",
+                                      "pv-path",
+                                      "pv-ip"
+                                    ]
+                                  }
+                                ]
+                              }
+                            },
+                            "required": [
+                              "pvc",
+                              "children"
+                            ]
+                          }
+                        ]
+                      }
+                    },
+                    "required": [
+                      "category",
+                      "type",
+                      "name",
+                      "children"
+                    ]
+                  }
+                ]
+              }
+            },
+            "required": [
+              "namespace_kube",
```

Comment:
verify what is always required (cluster without apps/pods, pods without pic, etc.)